app/templates/about_us.html

```diff
@@ -0,0 +1,101 @@
+                        <a class="nav-link" href="#">{{ gettext("Sign in") }}</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">{{ gettext("Sign up") }}</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="{{ url_for('agenda') }}">{{ gettext("Agenda") }}</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="{{ url_for('view_invitations') }}">Invitations</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/search">Search</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/celebrity">Celebrities Born Today</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="{{ url_for('salary_home') }}">Salary</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <br>
+    <div class="container">
+        <h1 class="moving-words">About The Project</h1>
+        <!--  credit for the animation goes to FrankieDoodie -->
+        <div>
+            <h3>Do you have a dream? How will you make it come true?</h3>
+            <p class="abouttext">
+                My dream will require some hard work. I know I MUST be:</p>
+            <ul>
+                <li> Organized</li>
+                <li>Well prepared</li>
+                <li>Plan ahead</li>
+            </ul>
+
+            <p class="abouttext">We started this project as a course project. Step by step it became our dream, this calendar is built by programmers. One of its many traits is the ability to add features.</p>
+            <ul>
+                <li> Want to monitor your expenses? We got it.</li>
+                <li>Want to organize a zoom meeting while still under quarantine? We got it.</li>
+            </ul>
+            <p class="abouttext">Even better - you got it! Develop your own feature and add it to the calendar. This calendar is open source developed mostly in Python as the finale to an extraordinary course. For the calendar development we selected features, aiming at required
+                and unique. Then each of us began the journey to implement it in the calendar. We worked collaboratively to put it together.
+                <br> if you like what you just read, please <a href="sign_in">sign in.</a>
+            </p>
+        </div>
+        <br><br><br><br>
+        <blockquote class="blockquote text-center">
+            <p class="mb-0"><i>“If you want to go fast, go alone. If you want to go far, go together.”</i></p><br>
+            <footer class="blockquote-footer"><cite title="Source Title">African proverb</cite></footer>
+        </blockquote>
+        <br><br><br><br>
```

Comment:
dont create white spaces as such. 
U can create <sections> tag and give it a:
```
margin-top: ..;
```

Reply:
changed!